functions/package.json

```diff
@@ -20,7 +20,8 @@
     "firebase-admin": "^9.1.0",
     "firebase-function-tools": "^1.1.3",
     "firebase-functions": "^3.9.1",
-    "glob": "^7.1.6"
+    "glob": "^7.1.6",
+    "inquirer": "^7.3.3"
```

Comment:
This package should be a dev dependency, right?

Reply:
Thought the same, but inquirer must be a dependency. Someone got the same issue [here](https://github.com/plopjs/node-plop/issues/117).